src/utils/private.ts

```diff
@@ -20,7 +20,13 @@ import {
   getRegisterWalletWithVerification,
 } from "./helpers";
 
-import DEFAULT_CONFIG_JSON from "../../ubiquibot-config-default.json";
+import fs from "fs";
+import stripJsonComments from "strip-json-comments";
+
+const configPath = "../../ubiquibot-config-default.jsonc";
+const configFile = fs.readFileSync(configPath, "utf8");
```

Comment:
As far as I remember if `fs` is used then the config file is not included in the netlify build, @0xcodercrane pls confirm

Reply:
We should avoid using `fs.readFileSync`.

`readFileSync` reads the config file in the execution step (means the config file is included when running) but to work as a serverless function, we should have all the necessary files included in the build step (exclude ubiquibot-config.yml -- it reads from the installed repo) because the ubiquibot GitHub app installed on the repo, not the source code being installed.